usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -6034,6 +6035,20 @@ CK_RV ep11tok_derive_key(STDLL_TokData_t *tokdata, SESSION *session,
         TRACE_ERROR("%s failedL hBaseKey=0x%lx\n", __func__, hBaseKey);
         return rc;
     }
+
+    rc = template_attribute_get_ulong(base_key_obj->template, CKA_KEY_TYPE, &keytype);
+    if (rc != CKR_OK) {
+        TRACE_ERROR("Invalid key type attribute\n");
+        goto error;
+    }
+
+    if (mech->mechanism == CKM_AES_XTS || keytype == CKK_AES_XTS) {
+        TRACE_ERROR("%s Key derive not supported\n",
```

Comment:
Please make the trace message a bit more procise, like 
"%s Key derive with AES-XTS is not supported\n"